<!DOCTYPE HTML>
<html>
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=gb2312" />
    <title>Research</title>
    <link href="Research.css" rel="stylesheet">
    <link href="font-awesome-4.3.0/css/font-awesome.min.css" rel="stylesheet">
	<link href="bootstrap-3.3.7-dist/css/bootstrap.min.css" rel="stylesheet">
	<link href="css2.css" rel="stylesheet">
  </head>
  
<body>
<!-- boostrap-iso begin -->
   <div class="boostrap-iso">
  <!--container begin-->
  <div id="container">
    
   
    <!--nav begin-->
    <div id="nav">
	    <div class="logo">
			<h4>MBB Lab</h4>
		</div>
		
		<div onclick="myDiv()" class="burger">
			<div class="line1"></div>
			<div class="line2"></div>
			<div class="line3"></div>
		</div>

		
         <ul id="nav-links">
           <li class="title1"><a href="Home.html">Home</a></li>
           <li class="title2"><a href="About MBB.html">About</a></li>
           <li class="title3"><a href="People.html">People</a>
              <ul>
                 <li><a href="subnav/People/PI.html">Principal Investigator</a></li>
                 <li><a href="subnav/People/Postdoctor.html">Postdoc</a></li>
                 <li><a href="subnav/People/PhD Student.html">PhD Student</a></li>
                 <li><a href="subnav/People/Master.html">Master Student</a></li>
                 <li><a href="subnav/People/Undergraduate.html">Undergraduate</a></li>
                 <li><a href="subnav/People/Technician.html">Technician</a></li>
                 <li><a href="subnav/People/Visiting Scholar.html">Visiting Scholar</a></li>
                 <li><a href="subnav/People/Alumni.html">Alumni</a></li>
              </ul>
           </li>
           <li class="title4"><a href="Research.html">Research</a>
              <ul>
                 <li><a href="subnav/Research/1.html">Sparse Model</a></li>
                 <li><a href="subnav/Research/2.html">Microscopic imaging</a></li>
                 <li><a href="subnav/Research/3.html">Biomedical imaging</a></li>
                 <li><a href="subnav/Research/4.html">Genomics Informatics</a></li>
                 <li><a href="subnav/Research/5.html">Imaging Genomics</a></li>
				 <li><a href="subnav/Research/6.html">devcog</a></li>
              </ul>
           </li>
           <li class="title5"><a href="Publications.html">Publications</a>
              <ul>
                 <li><a href="subnav/Publications/Book Chapters.html">Book Chapters</a></li>
                 <li><a href="subnav/Publications/Journal publications.html">Journal publications</a></li>
                 <li><a href="subnav/Publications/Conference papers.html">Conference papers</a></li>
              </ul>
           </li>
           <li class="title6"><a href="Software.html">Software</a></li>
           <li class="title7"><a href="Teaching.html">Teaching</a>
             <ul>
                 <li><a href="subnav/Teaching/Mathematical Modeling.html">Mathematical Modeling</a></li>
                 <li><a href="subnav/Teaching/Biomedical imaging.html">Biomedical imaging</a></li>
             </ul>
           </li>  
           <li class="title11"><a href="News.html">News</a></li>
           <li class="title0"><a href="Opportunity.html">Opportunity</a></li>
           <li class="title9"><a href="Contact.html">Contact</a></li> 
         </ul>
    </div><!--nav end-->
   </div>
<!--top-bottom1 end-->       
    
    
    <!--content begin-->
  <div class="sidebar">
   <div class="list-group">
     <a href="Research.html" class="list-group-item active">Research</a>
     <a href="subnav/Research/1.html" class="list-group-item">Sparse Model</a>
     <a href="subnav/Research/2.html" class="list-group-item">Microscopic imaging</a>
     <a href="subnav/Research/3.html" class="list-group-item">Biomedical imaging</a>
     <a href="subnav/Research/4.html" class="list-group-item">Genomics Informatics</a>
     <a href="subnav/Research/5.html" class="list-group-item">Imaging Genomics</a>
     <a href="subnav/Research/6.html" class="list-group-item">devcog</a>
  </div>
  </div>
   <div class="content1">
  
<h3 class="title">Research Description</h3> 
<div class="top">
<p>Our laboratory, Multiscale Bioimaging and Bioinformatics (MBB), has extensive experiences and expertise in biomedical imaging and bioinformatics at both the Biomedical Engineering department, and Biostatistics and Bioinformatics Department at both School of Sciences and Engineering and School of Public Health and Tropical Medicine. In addition, we are part of Tulane Center for Bioinformatics and Genomics (TBG), Tulane Cancer Center (TCC), Computer Science and Neurosciences Program. Our research group has been playing an active role in facilitating the collaboration between biomedical engineering and other departments in the campus.</p> 
<p>The following is an outline of our ongoing research projects, which span mathematics, computer sciences, statistics, biomedical engineering and health informatics.</p>
</div>

<div class="text1"><h3 class="title">1.Wavelets and sparse representations based signal processing</h3>
<p>Given our multi-disciplinary background in mathematical sciences, signal and image processing, biomedical imaging and bioinformatics, the representation and analysis of biomedical imaging and genomic data have been our long interest. We first proposed the <span class="bold">B-spline scale-space</span> theory as an alternative to the widely used Gaussian scale-space approach in the area of computer vision and image processing research. The theory we proposed has several advantages over traditional methods. We also develop a differential wavelets, which can have faster computational efficiency, which have found widespread applications in signal and image processing.</p>    
<div class="r1"><img src="pic/pic change/11.png" /></div>
<div class="list">
<p>a. Yu-Ping Wang, S. L. Lee, Scale-space derived from B-splines, IEEE Trans. Pattern Analysis and Machine Intelligence, vol. 20, no. 10, Oct. 1998, pp.1050-1065.</p> 
<p>b. Yu-Ping Wang, S. L. Lee, K. Torachi, Multiscale curvature based shape representation using B-spline wavelets, IEEE Trans. Image Processing, vol. 8, no 11, 1999. pp. 1586-1592. </p>
<p>c. Yu-Ping Wang, Image representations using multiscale differential operators, IEEE Trans. Image Processing, vol. 8, no. 12, 1999, pp. 1757-1771.</p>
<p>d. Yu-Ping Wang, Ruibin Qu. Fast implementation of scale-space by interpolatoy subdivision scheme, IEEE Trans. Pattern Analysis and Machine Intelligence, vol. 21, no. 9, 1999 pp.1050-1065.</p>
<p>Our recent effort has been the development of sparse representation models, where we propose a theory on an easy condition to recover the sparse signals. A number of papers are currently under review. </p> 
<p>e. J. Duan, C. Soussen, D. Brie, J. Idier and Y.-P. Wang, On LARS/homotopy equivalence conditions for  over-determined LASSO, IEEE Signal Processing Letters, 19(12), 2012.</p>
</div>
<p>The representation and analysis of multi-scale and multi-modal biological data sets is ubiquitous but challenging in many disciplines. We have been working on sparse data representation method, a hot topic in both signal processing, statistics and applied mathematics. The field is more exciting within the context of big data science and technique, which are highly promoted by NIH and NSF, as demonstrated by several program announcements recently. This is especially the case with multi-omics data, where huge amounts of genomic data of different nature, format, organization and structure are produced at different genomic platforms and at multiple scales. These data sets provide rich and complementary information; how to represent and integrate these muli-omics data for systems biology study has become a promising and emerging research topic. We continue this line of research and apply to both imaging and genomic data analysis.</p>
</div>


<div class="text2">
<h3 class="title">2.Microscopic image processing and industrial software development</h3>
<p>The PI, Dr. Wang, has many years of industrial experiences and made contributions to commercializing cytogenetic imaging analysis products. He has worked in bio-industry on various genomic imaging techniques including chromosome G-banding, FISH/M-FISH and microarray imaging under a number of NIH Small Business Innovation Research (SBIR) awards in collaboration with Texas Medical Center in Houston. He was the key team player in the development of <span class="bold">PowerGene</span><sup>TM</sup> chromosome karyotyping system and <span class="bold">PowerProbe</span><sup>TM</sup> FISH system that are now widely used in the market. </p>
<div class="list">
<p>a. Yu-Ping Wang, Q. Wu, Ken Castleman, and Z. Xiong, Chromosome Image Enhancement Using Multiscale Differential Operators, IEEE Trans. Medical Imaging, vol. 22, no.5, May, 2003.</p>
<p>b. Z. Liu, Z. Xiong, Q. Wu, Y. Wang, and K. Castleman, Cascaded differential and wavelet compression of chromosome images, IEEE Trans. on Biomedical Engineering, vol. 49, no. 4, 2002. </p>
<p>c. Yu-Ping Wang and Ken Castleman, Automated Registration of Multi-Color Fluorescence In Situ Hybridization (M-FISH) Images for Improving Color Karyotyping, Cytometry, Part A, 64A(2), April, 2005.</p>
</div>
<p>Given our experiences in developing industrial image processing software under SBIR research, we plan to 
<div class="r2"><img src="pic/pic change/22.png" /></div>
establish a close collaboration with the industry in the next few years so that some of our research results can be commercialized.</p>
</div>


<div class="text3">
<h3 class="title">3.Biomedical image processing</h3>
<p>The PI, Dr. Wang, started his career on <span class="bold">medical imaging</span> when working as a postdoc at Washington University Medical Center for cardiovascular imaging analysis with tagged MRI. He developed a technique to analyze the movement and deformation of the human heart for clinical diagnosis. Since then, we have made contributions to various biomedical imaging such as optical imaging, microscopic imaging and fMRI imaging, ranging from tissue and organ levels to cellular and molecular levels, as evidenced from my publications. </p>
<div class="list">
<p>a. Yu-Ping Wang, Y. Chen, A. A. Amini, Efficient LV Motion Estimation using Subspace Approximation Techniques, IEEE Trans. Medical Imaging, vol. 20, no.6, June 2001</p>
<p>b. Yu-Ping Wang, Y. Wang and P. Spencer, Fuzzy Clustering of Raman Spectral Imaging Data with a Wavelet-Based Noise Reduction Approach, Applied Spectroscopy, 60(7), 60(7), 2006. </p>
<p>c. Yu-Ping Wang, Husain Ragib, and Chi-Ming Huang, A wavelet approach for the identification of axonal synaptic varicosities from microscope images, IEEE Trans. Information Technology in Biomedicine, May, 11(7): 296-304, 2007.  </p>
<p>d. Hongbao Cao and Yu-Ping Wang, Segmentation of M-FISH images for improved classification of 
<div class="r3"><img src="pic/pic change/33.png" /></div>
chromosomes with an adaptive fuzzy c-means clustering algorithm, IEEE Trans. Fuzzy Systems, 2011, 20(1): 1-8.</p>
</div>
<p>We recently received an NSF EPSCoR project with New Mexico, Nebraska, and Louisiana to form a consortium on developmental chronnecto-genomics (Dev-CoG). The overarching goal is to advance understanding of childhood brain connectivity by developing new analytic approaches to study connectivity over brief and extended periods of time (the chronnectome), via multiple neuroimaging modalities. We will continue to develop multivariate approaches for multi-scale longitudinal modeling of developmental changes in multi-modal chronnectomic measures.</p>
</div>

<div class="text4">
<h3 class="title">4.Genomics Informatics or genomic signal processing</h3>
<p>Genomic data analysis in the context of post-genome era information processing is an emerging and exciting topic. In the last couple years, high resolution genetic probes have evolved from Human Genome Sequencing Project, which can detect subtle and cryptic genetic aberrations unobtainable with conventional probes, thus, holding great promise for personalized medicine. Among various projects we worked, we have made contributions to the detection of copy number variations (CNVs) first from microarray platform and currently from next generation sequencing (NGS) platform.  Recently, we proposed a novel statistical approach that can detect CNVs with better accuracy and more robustness. Our work has been <span class="bold">featured</span> in <a href="http://www.genengnews.com/gen-articles/cnv-strategies-get-a-rethink/5004/">Genetic Engineering & Biotechnology News</a> and <a href="http://www.biocompare.com/Editorial-Articles/156692-Find-All-the-Structural-Variants-with-These-Sequencing-based-CNV-Detection-Tools/">Biocompare</a> report.</p>
<div class="list">
<p>a. Chen J, Wang YP. A statistical change point model approach for the detection of DNA copy number variations in array CGH data. IEEE/ACM Trans Comput Biol Bioinform. 2009 Oct-Dec;6(4):529-41. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/19875853/">19875853</a>; PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4154476/">PMC4154476</a>.</p> 
<p>b. Y.-P.Wang, M. Gunampally, J. Chen, D. Bittel, M. Butler and W.-W. Cai, A Comparison of Fuzzy Clustering Approaches for Quantification of Microarray Gene Expression, Journal of VLSI Signal Processing Special Issue on Machine Learning for Microarray and Sequence Analysis, 50: 305-320, 2008.</p>
<div class="r4"><img src="pic/pic change/44.png" /></div>
<p>c. Duan J, Zhang JG, Deng HW, Wang YP. CNV-TV: a robust method to discover copy number variation from short sequencing reads. BMC Bioinformatics. 2013 May 2;14:150. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23634703/">23634703</a>; PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679874/">PMC3679874</a>. </p>
<p>d. Duan J, Deng HW, Wang YP. Common copy number variation detection from multiple sequenced samples. IEEE Trans Biomed Eng. 2014 Mar;61(3):928-37. PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4165854/">PMC4165854</a>.
</div> 
<p>We are currently focusing on two projects: a). Integration of multi-omics data; and b) the analysis of next generation sequencing data. Given the availability of vast amounts of data generated by a variety of high throughput genomic techniques, how to fuse the heterogeneous multimodal genomic information to improve the predication of cancer and the response to therapies is of tremendous significance in biomedicine.  We recently received a major NIH R01 award to continue this exciting research in collaboration with Tulane Center for Bioinformatics and Genomics (CBG) on the diagnosis of osteoporosis.</p>
</div>

<div class="text5">
<h3 class="title">5.Imaging Genomics</h3> 
<p>In the last few years, the combination of imaging and genetic approaches has become an emerging area in brain research, where multiple complementary data are utilized for systematic and comprehensive analysis of a patient. Our current effort is on multiscale and multimodal image and genomic <span class="bold">data integration</span>. We pioneered the use of sparse models based approaches for data integration, which have been supported by several NIH and NSF grants. We developed a sparse group CCA model, collaborative low rank regression model and sparse regressions for the correlation and integration of (epi)-genomic and multiple imaging data, and further link with biological knowledge bases for the diagnosis of complex diseases and detection of genes.</p>
<div class="list">
<p>a. Sheng J, Deng HW, Calhoun VD, Wang YP. Integrated analysis of gene expression and copy number data on gene shaving using independent component analysis. IEEE/ACM Trans Comput Biol Bioinform. 2011 Nov-Dec;8(6):1568-79. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/21519112/">21519112</a>; PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146966/">PMC3146966</a>. </p>
<p>b. Cao H, Duan J, Lin D, Calhoun V, Wang YP. Integrating fMRI and SNP data for biomarker identification for schizophrenia with a sparse representation based variable selection method. <br />BMC Med Genomics. 2013;6 Suppl 3:S2. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24565219/">24565219</a>; PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3980348/">PMC3980348</a>.</p>
<div class="r5"><img src="pic/pic change/55.png" /></div>
<p>c. Cao H, Duan J, Lin D, Shugart YY, Calhoun V, Wang YP. Sparse representation based biomarker selection for schizophrenia with integrated analysis of fMRI and SNPs. Neuroimage. 2014 Nov 15;102 Pt 1:220-8. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24530838/">24530838</a>; PubMed Central PMCID: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4130811/">PMC4130811</a>.</p>
<p>d.Yu-Ping Wang, Multiscale genomic imaging informatics, IEEE Signal Processing Magazine, Nov. Dec issue, pp. 169-172, 2009.</p>
</div>
<p>We currently have two ongoing R01 projects in collaboration with Prof. Vince Calhoun of Mind Research Network in Albuquerque. The project is about the integration of these multiscale and multimodal approaches, which has been promising for complex disease diagnosis and prognosis.</p>
</div>


<div class="text5">
<h3 class="title">6.Developmental Chronnecto-Genomics</h3> 
<p><span class="bold">Please check the following link:</span> <a href="http://devcog.mrn.org/" target="_blank">Developmental Chronnecto-Genomics</a></p>
</div>

</div>
    <!--content end-->
    
    
    <!--top-bottom2 begin-->   
   <!--copyright begin-->
    <div id="copyright">
        <div class="cr-top">
          <div class="map"><a href="https://www.google.com/maps/@29.9556836,-90.0712342,16z?hl=zh-CN" target="_blank">
          <i class="fa fa-map-marker"></i> Google map</a></div>
          <div class="E-mail"><a href="Contact.html"><i class="fa fa-envelope-o"></i> E-mail:</a><span class="e"> <a href="mailto:wyp@tulane.edu">wyp@tulane.edu</a></span></div>
          <div class="phone"><a href="Contact.html"><i class="fa fa-phone"></i> phone:</a><span class="tel"> 504-988-1341</span></div>
        </div>
        <div class="cr-bottom">Copyright <i class="fa fa-copyright"></i> <span id="year"></span> MBB Lab Research</div>     
    </div> 
    <!--copyright end --> 
  </div>
<!--  container end-->
<script src="nav.js"></script>
<script>
var d = new Date();
document.getElementById("year").innerHTML = d.getFullYear();
</script>
</div><!-- boostrap-iso end -->
</body>
</html>
<!--top-bottom2 end-->   
