<!DOCTYPE HTML>
<html>
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
	<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
	<title>Software</title>
    <link href="Software.css" rel="stylesheet">
    <link href="css2.css" rel="stylesheet">
    <link href="font-awesome-4.3.0/css/font-awesome.min.css" rel="stylesheet">
    <link href="bootstrap-3.3.7-dist/css/bootstrap.min.css" rel="stylesheet">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>
  
<body>
  <!--container begin-->
  <div id="container">
    
    
   
    <!--nav begin-->
    <div id="nav">
	    <div class="logo">
			<h4>MBB Lab</h4>
		</div>
		
		<div onclick="myDiv()" class="burger">
			<div class="line1"></div>
			<div class="line2"></div>
			<div class="line3"></div>
		</div>

		
         <ul id="nav-links">
           <li class="title1"><a href="Home.html">Home</a></li>
           <li class="title2"><a href="About MBB.html">About</a></li>
           <li class="title3"><a href="People.html">People</a>
              <ul>
                 <li><a href="subnav/People/PI.html">Principal Investigator</a></li>
                 <li><a href="subnav/People/Postdoctor.html">Postdoc</a></li>
                 <li><a href="subnav/People/PhD Student.html">PhD Student</a></li>
                 <li><a href="subnav/People/Master.html">Master Student</a></li>
                 <li><a href="subnav/People/Undergraduate.html">Undergraduate</a></li>
                 <li><a href="subnav/People/Technician.html">Technician</a></li>
                 <li><a href="subnav/People/Visiting Scholar.html">Visiting Scholar</a></li>
                 <li><a href="subnav/People/Alumni.html">Alumni</a></li>
              </ul>
           </li>
           <li class="title4"><a href="Research.html">Research</a>
              <ul>
                 <li><a href="subnav/Research/1.html">Sparse Model</a></li>
                 <li><a href="subnav/Research/2.html">Microscopic Imaging</a></li>
                 <li><a href="subnav/Research/3.html">Biomedical Imaging</a></li>
                 <li><a href="subnav/Research/4.html">Genomics Informatics</a></li>
                 <li><a href="subnav/Research/5.html">Imaging Genomics</a></li>
				 <li><a href="subnav/Research/6.html">devcog</a></li>
              </ul>
           </li>
           <li class="title5"><a href="Publications.html">Publications</a>
              <ul>
                 <li><a href="subnav/Publications/Book Chapters.html">Book Chapters</a></li>
                 <li><a href="subnav/Publications/Journal Publications.html">Journal Publications</a></li>
                 <li><a href="subnav/Publications/Conference Papers.html">Conference Papers</a></li>
              </ul>
           </li>
           <li class="title6"><a href="Software.html">Software</a>
           </li>
           <li class="title7"><a href="Teaching.html">Teaching</a>
             <ul>
                 <li><a href="subnav/Teaching/Mathematical Modeling.html">Mathematical Modeling</a></li>
                 <li><a href="subnav/Teaching/Biomedical Imaging.html">Biomedical Imaging</a></li>
             </ul>
           </li>  
           <li class="title11"><a href="News.html">News</a></li> 
           <li class="title0"><a href="Opportunity.html">Opportunity</a></li>
           <li class="title9"><a href="Contact.html">Contact</a></li> 
         </ul>
    </div><!--nav end-->
<!--top-bottom1 end-->     
    
   
    
    
    
    <!--content begin-->
    
     <!-- sidebar -->
     <div class="sidebar">
   <div class="list-group">
  <a href="Software.html" class="list-group-item active">Software</a>
  <a href="subnav/Software/gsCCA/gsCCA.html" class="list-group-item">gsCCA</a>
  <a href="subnav/Software/sparse representation based method/sparse representation based method.html" class="list-group-item">sparse representation based method</a>
  <a href="subnav/Software/(M-FISH) database/(M-FISH) database.html" class="list-group-item">(M-FISH) database</a>
  <a href="subnav/Software/CNV-TV/CNV-TV.html" class="list-group-item">CNV-TV</a>
  <a href="subnav/Software/sparse model/sparse model.html" class="list-group-item">sparse model</a>
  <a href="subnav/Software/Variant identification/Variant identification.html" class="list-group-item">Variant identification</a>
  <a href="subnav/Software/Fine mapping and test/Fine mapping and test.html" class="list-group-item">Fine mapping and test</a>
  <a href="subnav/Software/JCCA model/JCCA model.html" class="list-group-item">JCCA model</a>
  <a href="subnav/Software/FDRcorrectedSCCA/FDRcorrectedSCCA.html" class="list-group-item">FDRcorrectedSCCA</a>
  <a href="subnav/Software/Greedy projected distance correlation/Greedy projected distance correlation.html" class="list-group-item">Greedy projected distance correlation</a>
  <a href="subnav/Software/grpSLOPE/grpSLOPE.html" class="list-group-item">grpSLOPE</a>
  <a href="subnav/Software/RKUM/RKUM.html" class="list-group-item">RKUM</a>
  </div>
  </div>
  
    
    <div class="content1">
    
         <div class="content-top">
            <span class="ct1">Free software and data for public use</span><br />
            <span class="ct2">The software and data listed here is partially under NSF and NIH support.</span><br />
            <span class="ct3">Please acknowledge our publications when using our software.</span>
        </div>
        
        
        <div class="bs-callout"> <h4>Available Software:</h4></div>
        
        
        <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
        
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingOne">
      <h4 class="panel-title">
        <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
          Data integration with Group sparse Canonical Correlation Analysis (gsCCA)
        </a>
      </h4>
    </div>
    <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
      <div class="panel-body">
       <!--gsCCA start-->
       <div class="content-1">
            <p class="one">1. Data integration with <a href="https://sites.google.com/site/dongdongtonylin/software/software-for-group-sparse-canonical-correlation-analysis-group-sparse-cca" target="_blank">Group sparse Canonical Correlation Analysis</a> (gsCCA). | <a href="https://github.com/zbxnzs1990/Group-sparse-Canonical-Correlation-Analysis" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
                <p class="italic">Details can be found at:</p>
                <p class="comment">Dongdong Lin, Ji-Gang Zhang,  Li, Vince D. Calhoun, Hong-Wen Deng, Yu-Ping Wang: Group sparse canonical correlation analysis for genomic data integration. <a href="http://www.biomedcentral.com/1471-2105/14/245" target="_blank">BMC Bioinformatics 14</a> , highly accessed paper, 245 (2013)
                </p>
            </div>
        </div>
       <!--gsCCA end-->
       
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingTwo">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
          Imaging and genomic data integration with sparse representation based method
        </a>
      </h4>
    </div>
    <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
      <div class="panel-body">
      <!--sparse representation based method start-->
       <div class="content-1">
            <p class="two">2. Imaging and genomic data integration with <a href="http://hongbaocao.weebly.com/software-for-download.html" target="_blank">sparse representation based method</a>. | <a href="https://github.com/zbxnzs1990/sparse-representation-based-method" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>
            <p class="comment">H. Cao, J. Duan, D. Lin, YY Shugart, V. Calhoun,  and Y. Wang, Sparse Representation Based Biomarker Seleletion  for Schizophrenia with Integrated Analysis of fMRI and SNPs, NeuroImage (2014), http://dx.doi.org/10.1016j.neuroimage.2014.01.021</p>
            <p class="comment">H. Cao, J. Duan, D. Lin, V. Calhoun, and Y. Wang, Integrating fMRI and SNP data for biomarker identification for Schizophrenia with a sparse representation based variable selection method, BMC Medical Genomics, Nov.2013, 6(3):S2, doi:10.1186/1755-8794-6-S3-S2.</p>
            </div>
        </div>
      <!--sparse representation based method end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingThree">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
          Multi-color fluorescence in situ image (M-FISH) database
        </a>
      </h4>
    </div>
    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
      <div class="panel-body">
        <!--(M-FISH) database start-->
        <div class="content-1">
            <p class="three">3. Multi-color fluorescence in situ image <a href="https://sites.google.com/site/xiaobaocao006/database-for-download" target="_blank">(M-FISH) database</a>. | <a href="https://github.com/zbxnzs1990/M-FISH-Database" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>
            <p class="comment">J. Li, D. Lin, H. Cao, and Y. Wang, An improved sparse representation model with structural information for Multicolour Fluorescence In-Situ Hybridization (M-FISH) image classification, BMC Systems Biology, 2013, 7(4):S5  doi:10.1186/1752-0509-7-S4-S5</p>
            <p class="comment">H. Cao, H. Deng, Marilyn Li and, Y. Wang, Classification of Multicolor Fluorescence In-situ Hybridization (M-FISH) Images with Sparse Representation, IEEE Trans. Nanobioscience, 11(2):111-118, Jun. 2012.</p>
            </div>
        </div>
      <!--(M-FISH) database end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingFour">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
          Copy number variation detection from sequencing data using CNV-TV
        </a>
      </h4>
    </div>
    <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
      <div class="panel-body">
        <!--CNV-TV start-->
        <div class="content-1">
            <p class="four">4. Copy number variation detection from sequencing data using <a href="http://www.tulane.edu/~wyp/software/CNV_TV_pub.m" target="_blank">CNV-TV</a>. | <a href="https://github.com/zbxnzs1990/CNV-TV" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">J. Duan, J.-G. Zhang, H.-W. Deng, and Y.-P. Wang, "CNV-TV: A robust method to discover copy number variation from short sequencing reads," BMC Bioinformatics, highly accessed paper, vol. 14, pp. 1-12, 2013.</p>
            <div class="para"><p class="italic">Featured at:</p>
            <p>Genetic Engineering & Biotechnology News: "<a href="http://www.genengnews.com/gen-articles/cnv-strategies-get-a-rethink/5004/" target="_blank">CNV Strategies Get a Rethink</a>", Oct 1, 2013</p>
            <p>Biocompare: "<a href="http://www.biocompare.com/Editorial-Articles/156692-Find-All-the-Structural-Variants-with-These-Sequencing-based-CNV-Detection-Tools/" target="_blank">Never Miss a Variant Again with These Sequence-based CNV Detection Tools</a>",February 25, 2014</p></div>
            </div>
        </div>
      <!--CNV-TV end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingFive">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
          Copy number variation (CNV) detection from exome sequencing data using a sparse model
        </a>
      </h4>
    </div>
    <div id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive">
      <div class="panel-body">
       <!--sparse model start-->
       <div class="content-1">
            <p class="five">5. Copy number variation (CNV) detection from exome sequencing data using a <a href="http://www.tulane.edu/~wyp/software/Exon_CNV.m" target="_blank">sparse model</a>. | <a href="https://github.com/zbxnzs1990/sparse-model" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">J. Duan, M. Wan, H.-W. Deng, and Y.-P. Wang, "A sparse model based detection of copy number variations from exome sequencing data," IEEE Trans.</p> 
            <p class="comment">Biomedical Engineering, in review, 2015.</p>
            </div>
        </div>
      <!--sparse model end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingSix">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
          Variant identification from genome sequencing data (gsCCA)
        </a>
      </h4>
    </div>
    <div id="collapseSix" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSix">
      <div class="panel-body">
       <!--Variant identification start-->
        <div class="content-1">
            <p class="six">6. <a href="https://sites.google.com/site/shaolongscode/home/usr" target="_blank">Variant identification</a> from genome sequencing data (gsCCA). | <a href="https://github.com/zbxnzs1990/Variant-identification" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Shaolong Cao, Huaizhen Qin, Hong-Wen Deng and Yu-Ping Wang, A unified sparse representation for sequence variant identification for complex traits.</p>
            <p class="comment">Genetic epidemiology, 2014 Dec;38(8):671-9. doi: 10.1002/gepi.21849. Epub 2014 Sep 4.</p>
            </div>
        </div>
      <!--Variant identification end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingSeven">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSeven" aria-expanded="false" aria-controls="collapseSeven">
          Fine mapping and test for sequence association
        </a>
      </h4>
    </div>
    <div id="collapseSeven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSeven">
      <div class="panel-body">
      <!--Fine mapping and test start-->
      <div class="content-1">
            <p class="seven">7. <a href="https://sites.google.com/site/shaolongscode/home/uhdset" target="_blank">Fine mapping and test</a> for sequence association. | <a href="https://github.com/zbxnzs1990/Fine-mapping-and-test" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Shaolong Cao, Huaizhen Qin, Hong-Wen Deng and Yu-Ping Wang, A unified sparse representation for sequence variant identification for complex traits.</p>
            <p class="comment">Genetic epidemiology, 2014 Dec;38(8):671-9. doi: 10.1002/gepi.21849. Epub 2014 Sep 4.</p>
            </div>
        </div>
      <!--Fine mapping and test end-->
      </div>
    </div>
  </div>
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingEight">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseEight" aria-expanded="false" aria-controls="collapseEight">
          Joint CCA model for class-specific correlation analysis
        </a>
      </h4>
    </div>
    <div id="collapseEight" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEight">
      <div class="panel-body">
      <!--Joint CCA model start-->
      <div class="content-1">
            <p class="seven">8. <a href="https://sites.google.com/site/jianfang86/JSCCA" target="_blank"> Joint CCA model</a> for class-specific correlation analysis. | <a href="https://github.com/zbxnzs1990/Joint-CCA-model" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Jian Fang, Dongdong Lin, S. Charles Schulz, Zongben Xu, Vince D. Calhoun, Yu-Ping Wang. Joint sparse canonical correlation analysis for detecting differential imaging genetics modules.</p>
            <p class="comment">Bioinformatics (2016) 32 (22): 3480-3488. DOI: <a href="https://doi.org/10.1093/bioinformatics/btw485" target="_blank">https://doi.org/10.1093/bioinformatics/btw485</a>. Published: 27 July 2016. Article history</p>
            </div>
        </div>
      <!--Joint CCA model end-->
      </div>
    </div>
  </div>
  
    <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingNine">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseNine" aria-expanded="false" aria-controls="collapseNine">
          FDRcorrectedSCCA
        </a>
      </h4>
    </div>
    <div id="collapseNine" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingNine">
      <div class="panel-body">
      <!--FDRcorrectedSCCAl start-->
      <div class="content-1">
            <p class="seven">9. <a href="https://github.com/agisga/FDRcorrectedSCCA" target="_blank"> FDRcorrectedSCCA</a>. | <a href="https://github.com/agisga/FDRcorrectedSCCA" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Alexej Gossmann, Pascal Zille, V. D. Calhoun and Y.P. <span class="bold">Wang</span>, <a href="http://ieeexplore.ieee.org/document/8315045/" target="_blank">FDR-Corrected Sparse Canonical Correlation Analysis with Applications to Imaging Genomics</a>, Date of Publication: 13 March 2018, <span class="italic">IEEE Trans. Medical Imaging</span>, DOI: 10.1109/TMI.2018.2815583</p>
            </div>
        </div>
      <!--FDRcorrectedSCCA end-->
      </div>
    </div>
  </div>
  
	    <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingTen">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTen" aria-expanded="false" aria-controls="collapseTen">
          Greedy projected distance correlation (GDPC)
        </a>
      </h4>
    </div>
    <div id="collapseTen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTen">
      <div class="panel-body">
      <!--Greedy projected distance correlation (GDPC) start-->
      <div class="content-1">
            <p class="seven">10. <a href="https://sites.google.com/site/jianfang86/gpdc" target="_blank"> Greedy projected distance correlation (GDPC)</a>. | <a href="https://github.com/zbxnzs1990/Greedy-projected-distance-correlation-GDPC-" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
				<p class="italic">Details can be found at:</p>
                <p class="comment">Jian Fang, Chao Xu, Pascal Zille, Dongdong Lin, Hong-Wen Deng, Vince D. Calhoun, and Yu-Ping <span class="bold">Wang</span>, <a href="http://ieeexplore.ieee.org/document/8194926/" target="_blank">Fast and Accurate Detection of Complex Imaging Genetics Associations Based on Greedy Projected Distance Correlation</a>, IEEE Transactions on Medical Imaging, Dec.14, 2017, DOI: 10.1109/TMI.2017.2783244</p>
                </p>
            </div>
        </div>
      <!--Greedy projected distance correlation (GDPC) end-->
      </div>
    </div>
  </div>
  
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="heading11">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse11" aria-expanded="false" aria-controls="collapse11">
          Group sorted L1 penalized estimation (grpSLOPE) for feature selection
        </a>
      </h4>
    </div>
    <div id="collapse11" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading11">
      <div class="panel-body">
        <!--grpSLOPE start-->
        <div class="content-1">
            <p class="one">11.Group sorted L1 penalized estimation (<a href="https://github.com/agisga/grpSLOPEMC" target="_blank">grpSLOPE</a>) for feature selection. | <a href="https://github.com/agisga/grpSLOPEMC" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
                <p class="italic">Details can be found at:</p>
                <p class="comment">Alexej Gossmann, Shaolong Cao,  Damian Brzyski, Lan-Juan Zhao, Hong-Wen Deng, Yu-Ping <span class="bold">Wang, </span><a href="http://ieeexplore.ieee.org/document/8166765/" target="_blank">A sparse regression method for group-wise feature selection with false discovery rate control.</a> <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics,</i> 2017, Online ISSN: 1557-9964, Digital Object Identifier: 10.1109/TCBB.2017.2780106
                </p>
				</div>
            </div>
      <!--grpSLOPE end-->
      </div>
    </div>
  </div>
  
	<div class="panel panel-default">
    <div class="panel-heading" role="tab" id="heading12">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse12" aria-expanded="false" aria-controls="collapse12">
          RKUM
        </a>
      </h4>
    </div>
    <div id="collapse12" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading12">
      <div class="panel-body">
      <!--RKUM start-->
      <div class="content-1">
            <p class="seven">12. <a href="https://cran.r-project.org/web/packages/RKUM/index.html" target="_blank">RKUM</a>. | <a href="https://github.com/zbxnzs1990/RKUM-Robust-Kernel-Unsupervised-Methods" target="_blank"><strong>[GitHub link]</strong></a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Ashad Alam, Kenji Fukumizu, Yu-Ping <span class="bold">Wang</span>, <a href="https://www.sciencedirect.com/science/article/pii/S0925231218304211" target="_blank">Influence Function and Robust Variant of Kernel Canonical Correlation Analysis</a>, <span class="">Neurocomputing</span>, Volume 304, 23 August 2018, Pages 12-29, https://doi.org/10.1016/j.neucom.2018.04.008</p>
			
			<p class="comment">Ashad Alam, Vince D. Calhoun, Yu-Ping <span class="bold">Wang</span>, <a href="https://www.sciencedirect.com/science/article/pii/S0167947318300732" target="_blank">Identifying outliers using multiple kernel canonical correlation analysis with application to imaging genetics</a>, <span class="">Computational Statistics & Data Analysis</span>, September 2018, Volume 125, Pages 70-85, DOI: https://doi.org/10.1016/j.csda.2018.03.013</p>
            </div>
        </div>
      <!--RKUM end-->
      </div>
    </div>
  </div>

<!-- add software sidebar start
  <div class="panel panel-default">
    <div class="panel-heading" role="tab" id="headingFour">
      <h4 class="panel-title">
        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
          Copy number variation detection from sequencing data using CNV-TV
        </a>
      </h4>
    </div>
    <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
      <div class="panel-body">
        ##CNV-TV start
        <div class="content-1">
            <p class="four">4. Copy number variation detection from sequencing data using <a href="http://www.tulane.edu/~wyp/software/CNV_TV_pub.m" target="_blank">CNV-TV</a></p>
            <div class="sub-1">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">J. Duan, J.-G. Zhang, H.-W. Deng, and Y.-P. Wang, "CNV-TV: A robust method to discover copy number variation from short sequencing reads," BMC Bioinformatics, highly accessed paper, vol. 14, pp. 1-12, 2013.</p>
            <div class="para"><p class="italic">Featured at:</p>
            <p>Genetic Engineering & Biotechnology News: "<a href="http://www.genengnews.com/gen-articles/cnv-strategies-get-a-rethink/5004/" target="_blank">CNV Strategies Get a Rethink</a>", Oct 1, 2013</p>
            <p>Biocompare: "<a href="http://www.biocompare.com/Editorial-Articles/156692-Find-All-the-Structural-Variants-with-These-Sequencing-based-CNV-Detection-Tools/" target="_blank">Never Miss a Variant Again with These Sequence-based CNV Detection Tools</a>",February 25, 2014</p></div>
            </div>
        </div>
      ##CNV-TV end
      </div>
    </div>
  </div> 
  
 ##add software sidebar end -->
  
  
  
  <div class="bs-callout"> <h4>Our data is also listed</h4>
  <p><a class="active" href="https://www.nitrc.org/projects/gscca_2013/" target="_blank"><img src="pic/NITRC.png" class="NITRC"> Neuroimaging Informatics Tools and Resources Clearinghouse</a> is currently a free one-stop-shop collaboratory for science researchers that need resources such as neuroimaging analysis software, publicly available data sets, or computing power.</p>
  </div>
  
  
  
</div>


    </div>
    <!--content end-->
    
  <!--top-bottom2 begin-->   
   <!--copyright begin-->
    <div id="copyright">
        <div class="cr-top">
          <div class="map"><a href="https://www.google.com/maps/@29.9556836,-90.0712342,16z?hl=zh-CN" target="_blank">
          <i class="fa fa-map-marker"></i> Google map</a></div>
          <div class="E-mail"><a href="Contact.html"><i class="fa fa-envelope-o"></i> E-mail:</a><span class="e"> <a href="mailto:wyp@tulane.edu">wyp@tulane.edu</a></span></div>
          <div class="phone"><a href="Contact.html"><i class="fa fa-phone"></i> phone:</a><span class="tel"> 504-988-1341</span></div>
        </div>
        <div class="cr-bottom">Copyright <i class="fa fa-copyright"></i> <span id="year"></span> MBB Lab Research</div>     
    </div> 
    <!--copyright end --> 
  </div>
<!--  container end-->
<script>
var d = new Date();
document.getElementById("year").innerHTML = d.getFullYear();
</script>

<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="bootstrap-3.3.7-dist/js/bootstrap.min.js" type="text/javascript"></script>
	<script src="nav.js"></script>
</body>
</html>
<!--top-bottom2 end-->   
