<!DOCTYPE HTML>
<html>
  <head>
  <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
	<meta http-equiv="Content-Type" content="text/html; charset=gb2312">
	<title>Software</title>
    <link href="Software.css" rel="stylesheet">
    <link href="css2.css" rel="stylesheet">
    <link href="font-awesome-4.3.0/css/font-awesome.min.css" rel="stylesheet">
    <link href="css/bootstrap.min.css" rel="stylesheet">
  </head>
  
<body>
  <!--container begin-->
  <div id="container">
    
    
   
    <!--nav begin-->
    <div id="nav">
         <ul>
           <li class="title1"><a href="Home.html">Home</a></li>
           <li class="title2"><a href="About MBB.html">About</a></li>
           <li class="title3"><a href="People.html">People</a>
              <ul>
                 <li><a href="subnav/People/PI.html">Principal Investigator</a></li>
                 <li><a href="subnav/People/Postdoctor.html">Postdoc</a></li>
                 <li><a href="subnav/People/PhD Student.html">PhD Student</a></li>
                 <li><a href="subnav/People/Master.html">Master Student</a></li>
                 <li><a href="subnav/People/Undergraduate.html">Undergraduate</a></li>
                 <li><a href="subnav/People/Technician.html">Technician</a></li>
                 <li><a href="subnav/People/Visiting Scholar.html">Visiting Scholar</a></li>
                 <li><a href="subnav/People/Alumni.html">Alumni</a></li>
              </ul>
           </li>
           <li class="title4"><a href="Research.html">Research</a>
              <ul>
                 <li><a href="subnav/Research/1.html">Sparse Model</a></li>
                 <li><a href="subnav/Research/2.html">Microscopic Imaging</a></li>
                 <li><a href="subnav/Research/3.html">Biomedical Imaging</a></li>
                 <li><a href="subnav/Research/4.html">Genomics Informatics</a></li>
                 <li><a href="subnav/Research/5.html">Imaging Genomics</a></li>
				 <li><a href="subnav/Research/6.html">Idevcog</a></li>
              </ul>
           </li>
           <li class="title5"><a href="Publications.html">Publications</a>
              <ul>
                 <li><a href="subnav/Publications/Book Chapters.html">Book Chapters</a></li>
                 <li><a href="subnav/Publications/Journal Publications.html">Journal Publications</a></li>
                 <li><a href="subnav/Publications/Conference Papers.html">Conference Papers</a></li>
              </ul>
           </li>
           <li class="title6"><a href="Software.html">Software</a>
           </li>
           <li class="title7"><a href="Teaching.html">Teaching</a>
             <ul>
                 <li><a href="subnav/Teaching/Mathematical Modeling.html">Mathematical Modeling</a></li>
                 <li><a href="subnav/Teaching/Biomedical Imaging.html">Biomedical Imaging</a></li>
             </ul>
           </li>  
           <li class="title11"><a href="News.html">News</a></li> 
           <li class="title0"><a href="Opportunity.html">Opportunity</a></li>
           <li class="title9"><a href="Contact.html">Contact</a></li> 
         </ul>
    </div><!--nav end-->
<!--top-bottom1 end-->     
    
   
    
    
    
    <!--content begin-->
    
     <!-- sidebar -->
     <div class="sidebar">
   <div class="list-group">
  <a href="#" class="list-group-item list-group-item-info">Software</a>
  <a href="subnav/People/PI.html" class="list-group-item">gsCCA</a>
  <a href="subnav/People/Postdoctor.html" class="list-group-item">sparse representation based method</a>
  <a href="subnav/People/PhD Student.html" class="list-group-item">(M-FISH) database</a>
  <a href="subnav/People/Master.html" class="list-group-item">CNV-TV</a>
  <a href="subnav/People/Undergraduate.html" class="list-group-item">sparse model</a>
  <a href="subnav/People/Technician.html" class="list-group-item">Variant identification</a>
  <a href="subnav/People/Visiting Scholar.html" class="list-group-item">Fine mapping and test</a>
  <a href="subnav/People/Alumni.html" class="list-group-item">JCCA model</a>
  </div>
  </div>
    
    
    
    
    
    <div class="content1">
        <div class="content-top">
            <span class="ct1">Free software and data for public use</span><br />
            <span class="ct2">The software and data listed here is partially under NSF and NIH support.</span><br />
            <span class="ct3">Please acknowledge our publications when using our software.</span>
        </div>
        
        <div class="content-1">
            <p class="one">1. Data integration with <a href="https://sites.google.com/site/dongdongtonylin/software/software-for-group-sparse-canonical-correlation-analysis-group-sparse-cca" target="_blank">Group sparse Canonical Correlation Analysis</a> (gsCCA).</p>
            <div class="sub-1">
                <p class="italic">Details can be found at:</p>
                <p class="comment">Dongdong Lin, Ji-Gang Zhang,  Li, Vince D. Calhoun, Hong-Wen Deng, Yu-Ping Wang: Group sparse canonical correlation analysis for genomic data integration. <a href="http://www.biomedcentral.com/1471-2105/14/245" target="_blank">BMC Bioinformatics 14</a> , highly accessed paper, 245 (2013)
                </p>
            </div>
        </div>
        
        <div class="content-2">
            <p class="two">2. Imaging and genomic data integration with <a href="http://hongbaocao.weebly.com/software-for-download.html" target="_blank">sparse representation based method</a></p>
            <div class="sub-2">
            <p class="italic">Details can be found at:</p>
            <p class="comment">H. Cao, J. Duan, D. Lin, YY Shugart, V. Calhoun,  and Y. Wang, Sparse Representation Based Biomarker Seleletion  for Schizophrenia with Integrated Analysis of fMRI and SNPs, NeuroImage (2014), http://dx.doi.org/10.1016j.neuroimage.2014.01.021</p>
            <p class="comment">H. Cao, J. Duan, D. Lin, V. Calhoun, and Y. Wang, Integrating fMRI and SNP data for biomarker identification for Schizophrenia with a sparse representation based variable selection method, BMC Medical Genomics, Nov.2013, 6(3):S2, doi:10.1186/1755-8794-6-S3-S2.</p>
            </div>
        </div>
        
        <div class="content-3">
            <p class="three">3. Multi-color fluorescence in situ image <a href="https://sites.google.com/site/xiaobaocao006/database-for-download" target="_blank">(M-FISH) database</a></p>
            <div class="sub-3">
            <p class="italic">Details can be found at:</p>
            <p class="comment">J. Li, D. Lin, H. Cao, and Y. Wang, An improved sparse representation model with structural information for Multicolour Fluorescence In-Situ Hybridization (M-FISH) image classification, BMC Systems Biology, 2013, 7(4):S5  doi:10.1186/1752-0509-7-S4-S5</p>
            <p class="comment">H. Cao, H. Deng, Marilyn Li and, Y. Wang, Classification of Multicolor Fluorescence In-situ Hybridization (M-FISH) Images with Sparse Representation, IEEE Trans. Nanobioscience, 11(2):111-118, Jun. 2012.</p>
            </div>
        </div>
        
        <div class="content-4">
            <p class="four">4. Copy number variation detection from sequencing data using <a href="http://www.tulane.edu/~wyp/software/CNV_TV_pub.m" target="_blank">CNV-TV</a></p>
            <div class="sub-4">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">J. Duan, J.-G. Zhang, H.-W. Deng, and Y.-P. Wang, "CNV-TV: A robust method to discover copy number variation from short sequencing reads," BMC Bioinformatics, highly accessed paper, vol. 14, pp. 1-12, 2013.</p>
            <div class="para"><p class="italic">Featured at:</p>
            <p>Genetic Engineering & Biotechnology News: "<a href="http://www.genengnews.com/gen-articles/cnv-strategies-get-a-rethink/5004/" target="_blank">CNV Strategies Get a Rethink</a>", Oct 1, 2013</p>
            <p>Biocompare: "<a href="http://www.biocompare.com/Editorial-Articles/156692-Find-All-the-Structural-Variants-with-These-Sequencing-based-CNV-Detection-Tools/" target="_blank">Never Miss a Variant Again with These Sequence-based CNV Detection Tools</a>",February 25, 2014</p></div>
            </div>
        </div>
        
        <div class="content-5">
            <p class="five">5. Copy number variation (CNV) detection from exome sequencing data using a <a href="http://www.tulane.edu/~wyp/software/Exon_CNV.m" target="_blank">sparse model</a></p>
            <div class="sub-5">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">J. Duan, M. Wan, H.-W. Deng, and Y.-P. Wang, "A sparse model based detection of copy number variations from exome sequencing data," IEEE Trans.</p> 
            <p class="comment">Biomedical Engineering, in review, 2015.</p>
            </div>
        </div>
        
        <div class="content-6">
            <p class="six">6. <a href="https://sites.google.com/site/shaolongscode/home/usr" target="_blank">Variant identification</a> from genome sequencing data (gsCCA).</p>
            <div class="sub-6">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Shaolong Cao, Huaizhen Qin, Hong-Wen Deng and Yu-Ping Wang, A unified sparse representation for sequence variant identification for complex traits.</p>
            <p class="comment">Genetic epidemiology, 2014 Dec;38(8):671-9. doi: 10.1002/gepi.21849. Epub 2014 Sep 4.</p>
            </div>
        </div>
        
        <div class="content-6">
            <p class="seven">7. <a href="https://sites.google.com/site/shaolongscode/home/usr" target="_blank">Fine mapping and test</a> for sequence association.</p>
            <div class="sub-7">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Shaolong Cao, Huaizhen Qin, Hong-Wen Deng and Yu-Ping Wang, A unified sparse representation for sequence variant identification for complex traits.</p>
            <p class="comment">Genetic epidemiology, 2014 Dec;38(8):671-9. doi: 10.1002/gepi.21849. Epub 2014 Sep 4.</p>
            </div>
        </div>
        
        <div class="content-6">
            <p class="seven">8. <a href="https://sites.google.com/site/jianfang86/JSCCA" target="_blank"> Joint CCA model</a> for class-specific correlation analysis.</p>
            <div class="sub-7">
            <p class="italic">Details can be found at:</p>      
            <p class="comment">Jian Fang, Dongdong Lin, S. Charles Schulz, Zongben Xu, Vince D. Calhoun, Yu-Ping Wang. Joint sparse canonical correlation analysis for detecting differential imaging genetics modules.</p>
            <p class="comment">Bioinformatics (2016) 32 (22): 3480-3488. DOI: <a href="https://doi.org/10.1093/bioinformatics/btw485" target="_blank">https://doi.org/10.1093/bioinformatics/btw485</a>. Published: 27 July 2016. Article history</p>
            </div>
        </div>
        
    </div>
    <!--content end-->
    
  <!--top-bottom2 begin-->   
   <!--copyright begin-->
    <div id="copyright">
        <div class="cr-top">
          <div class="map"><a href="https://www.google.com/maps/@29.9556836,-90.0712342,16z?hl=zh-CN" target="_blank">
          <i class="fa fa-map-marker"></i> Google map</a></div>
          <div class="E-mail"><a href="Contact.html"><i class="fa fa-envelope-o"></i> E-mail:</a><span class="e"> <a href="mailto:wyp@tulane.edu">wyp@tulane.edu</a></span></div>
          <div class="phone"><a href="Contact.html"><i class="fa fa-phone"></i> phone:</a><span class="tel"> 504-988-1341</span></div>
        </div>
        <div class="cr-bottom">Copyright <i class="fa fa-copyright"></i> <span id="year"></span> MBB Lab Research</div>     
    </div> 
    <!--copyright end --> 
  </div>
<!--  container end-->
<script>
var d = new Date();
document.getElementById("year").innerHTML = d.getFullYear();
</script>
<script src="js/iquery.js"></script>
<script src="js/bootstrap.min.js"></script>
</body>
</html>
<!--top-bottom2 end-->   
